docs/github-connector/configuration.md

```diff
@@ -2,25 +2,32 @@
 
 ## Overview
 
-This document describes how to correctly connect GitHub repository to the GitHub Connector installed in Kyma environment. After completion you are able to handle events incoming from GitHub in lambdas.
+This document describes how to manually connect GitHub repository to existing GitHub Connector installed in Kyma enviroment. After completion you are able to handle events incoming from GitHub in lambdas.
+
 
 ## Prerequisites
 
 - Kyma with the GitHub Connector [installed](/docs/github-connector/installation.md)
-- Connection to Kyma
+- Connection to Kyma Console
 
 ## Installation
 
 1. Find the newly created GitHub Connector application and [bind it to the namespace](https://kyma-project.io/docs/components/application-connector/#tutorials-bind-an-application-to-a-namespace) of your choice.
 2. Open the settings of the GitHub repository you want to connect to, go to `Webhooks` page and click `Add webhook`.
 3. On the configuration page, fill the field `Payload URL` with exposed service URL (you can find it in `Kyma Console > {NAMESPACE} > APIs`) and add `/webhook` at the end of the URL.
-4. Set other fields as follows:
+4. To get Secret which is required during webhook setup, you need to use this comand:
+
+```shell
+kubectl get deployments -n {NAMESPACE} {DEPLOYMENT_NAME} -o jsonpath='{.spec.template.spec.containers[0].env[3].value}'
+```
+
+5. Set other fields as follows:
 
     - **Content type**: `application/json`
-    - **Secret**: `my-secret-key`
+    - **Secret**: previously obtained secret (nie wiem co z tym zrobic)
     - **SSL verification**: `Disabled`
 
-    >**NOTE:** Secret is defined statically in code and SSL verification is disabled.
+    >**NOTE:** If your Kyma cluster is not SSL secured, SSL verification will not be working.
```

Comment:
You say that SSL verification has to be disabled, then say it won't work if your cluster is not SSL secured. It's misleading. Specify if user can or cannot turn on the verification